templates/donation_successful.html

```diff
@@ -1,5 +1,194 @@
-<html>
+<!doctype html>
+<html lang="en" class="no-js">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+
+	<!-- Request Volkorn, Open Sans, Source Sans Pro, Signika (regular and semibold), Overlock -->
+	<link href='http://fonts.googleapis.com/css?family=Vollkorn|Open+Sans|Signika|Source+Sans+Pro:400,600,700' rel='stylesheet' type='text/css'>
+
+	<link href="https://fonts.googleapis.com/css?family=Overlock:400i" rel="stylesheet">
+
+
+	<link rel="stylesheet" href="../static/css/reset.css"> <!-- CSS reset -->
+	<link rel="stylesheet" href="../static/css/style.css"> <!-- Resource style -->
+	<link rel="stylesheet" href="../static/css/menu.css"> <!-- Resource style -->
+
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+
+	<!-- <link rel="stylesheet" href="../static/bower_components/font-awesome/css/font-awesome.min.css"> Resource style -->
+
+	<link rel="icon" type="image/x-icon" href="../static/favicon.ico"/>
+
+    <!--
+    <script>
+    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+    	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+    	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+    ga('create', '{{GA_ID}}', 'auto');
+    ga('send', 'pageview');
+    </script>
+-->
```

Comment:
I wasn't sure if you wanted to use the same GA_ID from the index.html file, since this is a different page. So, I just commented it out for now